### PR TITLE
#31 :arrow_up: update to parent version 10 (JDK 17, default Python fo…

### PR DESCRIPTION
**Pull request by d-ryan-ashcraft**
…r building to 3.11.9)

### CODE REVIEW

File: krausening-python/src/krausening/logging/log_manager.py [hunk 1]
```diff
@@ -5,16 +5,17 @@ class LogManager:
     """
     Class for handling logging.
     """
```
Comment (d-ryan-ashcraft) [inline krausening-python/src/krausening/logging/log_manager.py:7]: I (Informational): this came from running the latest version of Habushu on the existing codebase

File: krausening/src/test/java/org/technologybrewery/krausening/TestEncryptedKrausening.java [hunk 2]
```diff
@@ -28,6 +29,13 @@ public void testEncryptedPropertyReadProvidesDecryptedValue() {
 		krausening.loadProperties();
 		Properties properties = krausening.getProperties(ENCRYPTED_PROPERTIES);
 		assertNotNull(properties);
```
Comment (d-ryan-ashcraft) [inline krausening/src/test/java/org/technologybrewery/krausening/TestEncryptedKrausening.java:31]: I: the lamba breaks things with the encryptor (and arguably shouldn't be in this imperative codebase anyway)

**Review by csun-cpointe — APPROVED** [review]

File: krausening-python/pom.xml [hunk 3]
```diff
@@ -18,6 +18,9 @@
             <plugin>
                 <groupId>org.technologybrewery.habushu</groupId>
                 <artifactId>habushu-maven-plugin</artifactId>
+                <configuration>
+                    <pythonVersion>${version.python.default}</pythonVersion>
+                </configuration>
             </plugin>
```
Comment (meliz19) [inline krausening-python/pom.xml:24]: Q: Will this change the value of the current version of the project in the [pyproject.toml](https://github.com/TechnologyBrewery/krausening/blob/d297c64416d4a72d6352d5983a88a61804207d33/krausening-python/pyproject.toml#L3C1-L3C19)?

Comment (d-ryan-ashcraft) [reply krausening-python/pom.xml:24]: No - this is the version that we'll use to build only.

**Pull request merged**